CHANGELOG.md

```diff
@@ -4,6 +4,99 @@
 -   Upgrade `keycloak-js` to v26
 -   Remove prop-types
 
+**Breaking changes**
+
+The setup has been drastically simplified:
+
+```diff
+-import React, { useState, useRef, useEffect } from 'react';
+import React from 'react';
+import { Admin } from 'react-admin';
+import simpleRestProvider from 'ra-data-simple-rest';
+import Keycloak, {
+    KeycloakConfig,
+    KeycloakTokenParsed,
+    KeycloakInitOptions,
+} from 'keycloak-js';
+import { keycloakAuthProvider, httpClient } from 'ra-keycloak';
+
+const config: KeycloakConfig = {
+    url: '$KEYCLOAK_URL',
+    realm: '$KEYCLOAK_REALM',
+    clientId: '$KEYCLOAK_CLIENT_ID',
+};
+
+const initOptions: KeycloakInitOptions = { onLoad: 'check-sso' };
```

Comment:
The default was `{ onLoad: 'login-required' }` in the previous version. Encouraging moving to `{ onLoad: 'check-sso' }` is a good move IMO, but then you require the `LoginPage` too.

```suggestion
-const initOptions: KeycloakInitOptions = { onLoad: 'login-required' };
+const initOptions: KeycloakInitOptions = { onLoad: 'check-sso' };
```

Reply:
Not really needed IMO

Reply:
Yes it was in the documetation but it wasn't required